.github/workflows/main.yml

```diff
@@ -113,9 +113,10 @@ jobs:
     needs: [ pytest-common, pytest-tf, pytest-torch ]
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/download-artifact@v3
+      - uses: actions/download-artifact@v4
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
         with:
           flags: unittests
           fail_ci_if_error: true
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This secret does not exist. What should I do to get one ?

Reply:
@odulcy-mindee login at https://about.codecov.io/ and copy the token  (repos -> doctr -> settings)  the sec token is it which needs to be added to the repo secrets :)

Reply:
done